bootstrap/Dockerfile

```diff
@@ -82,4 +68,4 @@ ENV GROUP_ID 1000
 ENV GROUP kubeflow
 
 WORKDIR /opt/kubeflow
-ENTRYPOINT ["/opt/kubeflow/start.sh"]
+CMD ["/opt/kubeflow/start.sh"]
```

Comment:
Why command and not entrypoint?

Reply:
Want to make it optional as we don't need it when deploy bootstrapper.yaml.